### PR TITLE
Add operationId properties to endpoints

### PR DESCRIPTION
**Pull request by DonMartin76**
- This enables client SDK generators to create nicer names for the functions
- Otherwise auto generated names are used, which are mostly suboptimal

This would generally make it easier to use SDK generation tools such as [OpenAPI Generator](https://github.com/OpenAPITools/openapi-generator) to generate client SDKs for the HTTP API.

This PR only contains a change to the OpenAPI spec, and it only adds `operationId` properties. These are typically not show in e.g. SwaggerUI or ReDoc output, but can be leveraged by e.g. above-mentioned tools.

Names/operation IDs are of course up for discussion; I picked some which looked as if they would make sense.

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by akelad** [comment]
Thanks for submitting this PR, we'll give it a review as soon as possible

File: docs/_static/spec/rasa.yml [hunk 1]
```diff
@@ -398,6 +409,7 @@ paths:
       security:
       - TokenAuth: []
       - JWT: []
+      operationId: evaluateModelStories
```
Comment (tabergma) [inline docs/_static/spec/rasa.yml]: To keep it consistent with the operation id of `/model/test/intents`, I would use
```suggestion
      operationId: testModelStories
```

**Comment by wochinge** [comment]
@tabergma What's the state here?

**Comment by tabergma** [comment]
Ready to be merged. Sorry.

**Pull request merged**